app/controllers/users/my_rooms_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Users::MyRoomsController < ApplicationController
+  def index
+    @reviewer_rooms =
+      case params[:open_state]
+      when 'closed'
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.


<!-- Comment by [Sider](https://sider.review) -->